assets/src/js/header.js

```diff
@@ -93,6 +98,20 @@ export const setupHeader = function($) {
     $('.nav-menu-toggle').trigger('click');
```

Comment:
Is this maybe related to the double click you mention in the PR description? I think it's best to avoid triggering the click event just because the handler has the logic we want to execute. We can instead put that logic in a function and call it from multiple places.

Reply:
No the double click comes from the closeInactiveMenus() function clicking on everything and the fact that there's 2 different search toggles controlling the target, so a click on each toggle but then a revert in the effect on the target